src/react-vite/12-testing/testing.md

```diff
@@ -38,71 +38,77 @@ React Testing Library’s north star is:
 
 > The more your tests resemble the way your software is used, the more confidence they can give you.
 
-This leads to three [Guiding Principles](https://testing-library.com/docs/guiding-principles/):
-
-> 1. If it relates to rendering components, then it should deal with DOM nodes rather than component instances, and it should not encourage dealing with component instances.
-
 React Testing Library encourages tests that focus on how the user interacts with the application,
-rather than the implementation details of the components. This approach makes the tests more
-resilient to changes in the application’s codebase, as they are less likely to break with
-refactoring if the user interface and behavior remain consistent.
-
-> 2. It should be generally useful for testing the application components in the way the user would use it. We are making some trade-offs here because we’re using a computer and often a simulated browser environment, but in general, utilities should encourage tests that use the components the way they're intended to be used.
-
-The library promotes accessibility in React applications by providing tools and encouraging patterns
-that make it easier to create accessible web interfaces. By using semantic HTML and ARIA roles in
-tests, developers are nudged towards building applications that are more accessible to all users,
-including those using assistive technologies.
-
-> 3. Utility implementations and APIs should be simple and flexible.
-
-React Testing Library is designed to be lightweight and straightforward, avoiding the complexity that
-can sometimes come with extensive testing frameworks. It encourages good testing practices, such as
-avoiding testing the internal state of components and instead focusing on what the user would see
-and do. The library’s API is intentionally limited to encourage tests that align with these principles.
```

Comment:
A lot of this was lifted from the intro RTL training we gave in the fall: https://bitovi.atlassian.net/wiki/spaces/COMPENDIUM/pages/472449062/Practical+Unit+Testing+with+Jest+and+React+Testing+Library#Intro

Slides here: https://bitovi.atlassian.net/wiki/spaces/COMPENDIUM/pages/472449062/Practical+Unit+Testing+with+Jest+and+React+Testing+Library#Presentation

I just want to double check that we do want to remove any explanation of RTL from Academy, or if we want to leave it in Academy but either summarize or remove it from the slides (it’s ok for us to have more info in Academy than we cover in the slides).

Reply:
It seems like a lot of detailed explanation that isn't in other modules. Does @christopherjbaker have an opinion?